ember_debug/libs/promise-assembler.js

```diff
@@ -93,7 +93,7 @@ class PromiseAssembler extends BaseObject {
     if (guid) {
       const index = this.promiseIndex[guid];
       if (index !== undefined) {
-        return this.all.get(index);
+        return this.all[index];
```

Comment:
Are we sure this is never a proxy and won't need `get` ever? Could we add a small test to prevent regressions here?

Reply:
Yes. I missed this during the migration to native array wl